jupyter_server/auth/identity.py

```diff
@@ -81,6 +81,39 @@ def fill_defaults(self):
         if not self.display_name:
             self.display_name = self.name
 
+    def to_string(self) -> str:
+        """Serialize a user to a string for storage in a cookie
+        If overriding in a subclass, make sure to define from_string as well.
```

Comment:
Let's not assume this is for cookie usage...
```suggestion
        """Serialize a user to a string
        
        If overriding in a subclass, make sure to define the class method from_string as well.
```